lib/requester/browser/request.js

```diff
@@ -128,7 +128,11 @@ function request(options, callback) {
         }
         if((new RegExp(METHODS_WITH_BODY.join('|'), 'i')).test(options.method)){
             var encoding = (options.encoding || 'application/x-www-form-urlencoded').toLowerCase();
-            options.headers['content-type'] = encoding;
+
+            if (!options.headers['content-type'] && !options.headers['Content-Type']) {
+                options.headers['content-type'] = encoding;
+            }
+
```

Comment:
It could be made stricter similar to this:-
https://github.com/request/request/blob/bfb3a469a1c51ebf6a3c1f4be553b43eaafcf47c/request.js#L1137